### DISCUSSION TITLE
Iam not able to change my tauri application icon

### DISCUSSION THREAD

**Discussion by AkaashOruvan** [tauri-apps/tauri#9109] [General]
![Screenshot from 2024-03-07 11-00-45](https://github.com/tauri-apps/tauri/assets/113968065/5a1f7cf6-7028-4690-b265-9c063e5837a9)

I followed all the ways the youtube and in the documentation. the png files modified in the icons folder but still the changes are not reflected. Im using a linux machine.

**Comment by FabianLars** [comment]
I may be incorrect but i think gnome only loads icons from .desktop files in which case `tauri dev` won't be able to display the icon. YInstead actually building the app and installing the .deb (if you don't use debian/ubuntu you may have to convert that to an installer for your distro, there are usually tools for that) should work.
The appimage also won't show the icon then though.

**Reply by zsdsys** [reply]
I am still seeing this problem on Debian.  I am generating the icons with cargo tauri icon.  The .desktop file makes no difference, the shortcut icon works fine/is correct from that but the running application still has the "default icon".  The icon works on all other platforms without issue, this is specifically on debian/linux.
